.github/workflows/build.yml

```diff
@@ -0,0 +1,119 @@
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on: push
```

Comment:
Do we really want it on push? This will try to create run this for every branch in every forks...
So in my local fork, it will try (without success) to push to test-pypi every time I do a `git push`...

What happen with testpypi? With each merged PR it will create a new release. But we don't update the version number. Not sure how it handles it.